packages/squiggle-lang/src/PointSet/PointSetDist_Scoring.ts

```diff
@@ -53,49 +48,14 @@ export const WithDistAnswer = {
     estimate: AnyPointSet;
     answer: AnyPointSet;
   }): Result.result<number, OperationError> {
-    const combineAndIntegrate = (estimate: AnyPointSet, answer: AnyPointSet) =>
-      Result.fmap(
-        combinePointwise(estimate, answer, WithDistAnswer.integrand),
-        (t) => t.integralSum()
-      );
-
-    const getMixedSums = (
-      estimate: AnyPointSet,
-      answer: AnyPointSet
-    ): Result.result<[number, number], OperationError> => {
-      const esti = estimate.toMixed();
-      const answ = answer.toMixed();
-      const estiContinuousPart = esti.toContinuous();
-      const estiDiscretePart = esti.toDiscrete();
-      const answContinuousPart = answ.toContinuous();
-      const answDiscretePart = answ.toDiscrete();
-      if (
-        estiContinuousPart &&
-        estiDiscretePart &&
-        answContinuousPart &&
-        answDiscretePart
-      ) {
-        return Result.merge(
-          combineAndIntegrate(estiDiscretePart, answDiscretePart),
-          combineAndIntegrate(estiContinuousPart, answContinuousPart)
-        );
-      } else {
-        return Result.Err(new OtherOperationError("unreachable state"));
-      }
-    };
-
-    if (
-      (isContinuous(estimate) && isContinuous(answer)) ||
-      (isDiscrete(estimate) && isDiscrete(answer))
-    ) {
-      return combineAndIntegrate(estimate, answer);
-    } else {
-      return Result.fmap(
-        getMixedSums(estimate, answer),
-        ([discreteSamples, continuousSamples]) =>
-          discreteSamples + continuousSamples
-      );
-    }
+    return Result.fmap(
+      Mixed.combinePointwise(
+        estimate.toMixed(),
+        answer.toMixed(),
+        WithDistAnswer.integrand
+      ),
+      (t) => t.integralSum()
+    );
```

Comment:
Table in https://squigglehub.org/users/berekuk/models/scoring-experiments checks that this gives the same results; also all tests are still passing.

But I haven't checked very carefully that the benavior is still the same in _all_ cases; if there are any new bugs, then `Mixed` is to blame here.Having less duplicate code is more important, less risk for errors in the future.

Also, we discussed on Discord how maybe we shouldn't allow scoring on mixed dists, but I haven't changed that here.